lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -816,16 +816,18 @@ def cmd_transport(*args)
   end
 
   @@migrate_opts = Rex::Parser::Arguments.new(
-    '-p'  => [true,  'Writable path - Linux only (eg. /tmp).'],
-    '-t'  => [true,  'The number of seconds to wait for migration to finish (default: 60).'],
-    '-h'  => [false, 'Help menu.']
+    '-P' => [true, 'PID to migrate to.'],
```

Comment:
Given that so many of us are now geared to `migrate <pid>` should we no default to this anyway and just handle cases where it's not given and look for a process name instead when `-N` is given?

My guess is that using `migrate -P <pid>` is going to feel like moving from Windows 7 to Windows 8 and not having a start menu any more. I'd rather see it behave as it did, but have the new parameter for string-based process names.

Thoughts?